ambry-cloud/src/main/java/com/github/ambry/cloud/CloudContainerCompactor.java

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright 2020 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.cloud;
+
+
+/**
+ * Class that runs scheduled or on-demand compaction of blobs of deprecated containers.
+ */
+public class CloudContainerCompactor {
+
```

Comment:
Class isn't referenced anywhere.

Reply:
It was going to be reference in the next PR, but have removed it for now.